frontend/common/src/messages/index.ts

```diff
@@ -1,3 +1,4 @@
+import { type SocketMessageType } from '@/types/websocket-messages';
```

Comment:
ok, so this is the reason for the failure this type is not actually available in the common library since it comes from the app itself.

You must move the type from the app to the common if you want to use it.

Reply:
alrighty, checking... thanks.

Reply:
it'll take a lot to move the definition of SocketMessageType to the common lib